tests/fixtures.py

```diff
@@ -778,12 +780,92 @@ async def _sock_from_other_organization_factory(
     return _sock_from_other_organization_factory
 
 
-@pytest.fixture
-def mocked_parsec_ext_smartcard(monkeypatch):
+def create_test_certificates(tmp_path):
+    """
+    Requires openssl to be installed. Only used on linux when parsec_ext is installed.
+    """
+    # Create openssl config
+    (tmp_path / "openssl.cnf").write_text("keyUsage = digitalSignature")
+
+    # OpenSSL commands for creating the CA key, the CA certificate, the client key and the client certificate.
+    commands = """\
+openssl req -x509 -newkey rsa:4096 -keyout ca_key.pem -out ca_cert.pem -sha256 -days 365 -subj "/CN=My CA" -addext "keyUsage = digitalSignature" -passout pass:test
+openssl req -out client_request.csr -newkey rsa:4096 -keyout client_key.pem -new -passout pass:test -subj "/CN=John Doe/emailAddress=john@example.com"
+openssl x509 -req -days 365 -in client_request.csr -CA ca_cert.pem -CAkey ca_key.pem -passin pass:test -CAcreateserial -out client_cert.pem -extfile "./openssl.cnf"
+"""
+
+    # Run the commands
+    for command in commands.splitlines():
+        subprocess.run(command, shell=True, check=True, cwd=tmp_path)
+
+    # Concatenate the key and the certificate
+    combined_data = (tmp_path / "client_cert.pem").read_bytes() + (
+        tmp_path / "client_key.pem"
+    ).read_bytes()
+    (tmp_path / "client_combined.pem").write_bytes(combined_data)
+
+    # Return certificates and password
+    return tmp_path / "ca_cert.pem", tmp_path / "client_combined.pem", "test"
```

Comment:
don't want to use `P@ssw0rd` for password ? (`test` seems too generic when debugging)

Reply:
4a1617256